### ISSUE TITLE
formula notation not correct

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/pyopenms-docs#360]
**Describe the problem you encountered**

![image](https://user-images.githubusercontent.com/5803621/221518157-ce729251-9da1-4100-b517-380cdf58e048.png)

should be something like [2M+H]^{2+}

**Documentation version**

**Comment by jpfeuffer** [comment]
See also my comments: https://github.com/OpenMS/pyopenms-docs/pull/331#discussion_r1109580670
please double-check if they all were fixed.

**Comment by jpfeuffer** [comment]
And no, it should not be `[2M+H]^{2+}` but `[M + 2H]^{2+}` . Please note the spaces and the fact that it is not two times the precursor but the precursor with two additional hydrogens.

**Comment by timosachsenberg** [comment]
In pyopenms-next?

**Comment by jpfeuffer** [comment]
What is pyopenms-next?

**Comment by timosachsenberg** [comment]
In :latest the error is still there.
Pyopenms doc version

**Comment by timosachsenberg** [comment]
Filtering Mass Spectra by :term`MS` Level[](https://pyopenms.readthedocs.io/en/latest/ms_data.html#filtering-mass-spectra-by-term-ms-level)

maybe term is not compatible with subsections

**Comment by jpfeuffer** [comment]
No, it is an Error: https://github.com/OpenMS/pyopenms-docs/blob/0068b5c0d20fd00bb8f811542bc907e0911a3dad/docs/source/ms_data.rst?plain=1#L642

**Comment by timosachsenberg** [comment]
is this done?
